### PR TITLE
Feature/send component info over grpc

### PR DESCRIPTION
**Pull request by mksm20**
Added Specific Component to Refinement, consistency and determinism failure - expanded simple id. tostring and created a getter function for component_id

### CODE REVIEW

File: src/TransitionSystems/location_id.rs [hunk 1]
```diff
@@ -110,6 +110,17 @@ impl LocationID {
             LocationID::AnyLocation() => true,
         }
     }
+    pub fn get_component_id(&self) -> Option<String> {
```
Comment (S3JER) [inline src/TransitionSystems/location_id.rs:113]: Mangler dokumentation ;)

**Review by S3JER — CHANGES_REQUESTED** [review]

File: src/TransitionSystems/location_id.rs (hunk 1 above)
Comment (mksm20) [reply src/TransitionSystems/location_id.rs:115]: er tilføjet

**Review by WassawRoki — APPROVED** [review]
I mean looks fine, men hvad sker der for github actions?

**Comment by S3JER** [comment]
I skal lige update jeres branch fra main igen, ellers ser det fint ud

**Review by S3JER — APPROVED** [review]

**Pull request merged**